styles/style.css

```diff
@@ -17,6 +17,131 @@
   padding: 0;
 }
 
+/*start-page*/
+.start-page {
+  min-height: 100vh;
+  background: url("../img/castelmezzano-1979546 1.svg") center no-repeat;
+  background-size: cover;
+  padding: 51px 104px 0 104px;
+  margin: 0 auto;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  height: 46px;
+  margin-bottom: 196px;
+  box-sizing: border-box;
+}
+
+.logo-img {
+  width: 205px;
+  height: 40px;
+}
+
+.nav {
+  display: flex;
+  align-items: center;
+}
+
+.nav-link {
+  justify-content: center;
+  text-decoration: none;
+  font-family: 'Roboto', sans-serif;
+  font-weight: normal;
+  font-size: 24px;
+  line-height: 1.2em;
+  color: #FFFFFF;
+  margin-right: 24px;
+  padding-bottom: 12px;
+}
+
+.nav-link:hover {
+  border-bottom: 4px solid #F5BD41;
+}
+
+.btn {
+  border: none;
+  background-color: #121212;
+  margin-left: 24px;
+}
+
+.main-title {
+  font-family: "Roboto", sans-serif;
+  font-size: 50px;
+  font-weight: 500;
+  line-height: 1.2em;
+  color: #ffffff;
+  margin-bottom: 100px;
+}
+
+.form-search{
+  display: flex;
+  max-height: 64px;
+  box-sizing: border-box;
+  margin-bottom: 129px;
+}
```

Comment:
How about `background-color: #fff`
It looks a bit messy now
<img width="1151" alt="Screenshot 2022-03-11 at 19 19 19" src="https://user-images.githubusercontent.com/25488411/157906582-b26f1cec-449b-4b0a-bcc7-8c500fe5ac57.png">